lib/flipper/adapters/redis.rb

```diff
@@ -26,15 +26,33 @@ def features
         read_feature_keys
       end
 
+      def self.redis_sadd_returns_boolean?
+        return @redis_sadd_returns_boolean if defined?(@redis_sadd_returns_boolean)
+        @redis_sadd_returns_boolean = true
+          if ::Redis.respond_to?(:sadd_returns_boolean)
```

Comment:
it will return `true` on the second and so on call.

You probably meant

```suggestion
        @redis_sadd_returns_boolean =
          if ::Redis.respond_to?(:sadd_returns_boolean)
```

Reply:
Lol, yes. That's embarrassing